test/powershell/Modules/PSReadLine/PSReadLine.Tests.ps1

```diff
@@ -42,6 +42,13 @@ Describe "PSReadLine" -tags "CI" {
         (Get-PSReadLineKeyHandler | where { $_.Key -ceq "Alt+B" }).Function | Should Be SelectBackwardWord
     }
 
+    It "Should ignore case when using Function binding" {
+        Set-PSReadlineOption -EditMode Windows
+        $lowerCaseFunctionName = "yank";
+        Set-PSReadlineKeyHandler "Ctrl+k" -Function $lowerCaseFunctionName
```

Comment:
Save/restore is not always possible - if a key was bound to a script block, there is currently no way to get the script block.

I think it's reasonable to use some key like `F24`.